tools/defaults.bzl

```diff
@@ -40,3 +43,73 @@ def ts_library(
         # @external_end
         **kwargs
     )
+
+def pkg_npm(name, use_prodmode_output = False, **kwargs):
+    """Default values for pkg_npm"""
+    visibility = kwargs.pop("visibility", None)
+
+    common_substitutions = dict(kwargs.pop("substitutions", {}))
+    substitutions = dict(common_substitutions, **{
+        # TODO: Current build script relies on 0.0.0 in package.json; uncomment after repalcing build script.
+        #"0.0.0-PLACEHOLDER": "0.0.0",
+    })
+    stamped_substitutions = dict(common_substitutions, **{
+        # TODO: Current build script relies on 0.0.0 in package.json; uncomment after repalcing build script.
+        #"0.0.0-PLACEHOLDER": "{BUILD_SCM_VERSION}",
+        "0.0.0": "{BUILD_SCM_VERSION}",
+    })
```

Comment:
I commented this out for now, but I thought it would be a good idea to use `0.0.0-PLACEHOLDER` rather than the current `0.0.0` versions in each package's `package.json` file.` I can't change those package.json files yet as that would break the current build script that uses 0.0.0 for version substitution.